Emby.Server.Implementations/Devices/DeviceManager.cs

```diff
@@ -194,172 +170,6 @@ private string GetDevicePath(string id)
-
-            _libraryMonitor.ReportFileSystemChangeBeginning(path);
-
-            try
-            {
-                using (var fs = new FileStream(path, FileMode.Create, FileAccess.Write, FileShare.Read))
-                {
-                    await stream.CopyToAsync(fs).ConfigureAwait(false);
-                }
-
-                AddCameraUpload(deviceId, file);
-            }
-            finally
-            {
-                _libraryMonitor.ReportFileSystemChangeComplete(path, true);
-            }
-
-            if (CameraImageUploaded != null)
-            {
-                CameraImageUploaded?.Invoke(this, new GenericEventArgs<CameraImageUploadInfo>
-                {
-                    Argument = new CameraImageUploadInfo
-                    {
-                        Device = device,
-                        FileInfo = file
-                    }
-                });
-            }
-        }
-
-        private void AddCameraUpload(string deviceId, LocalFileInfo file)
-        {
-            var path = Path.Combine(GetDevicePath(deviceId), "camerauploads.json");
-            Directory.CreateDirectory(Path.GetDirectoryName(path));
-
-            lock (_cameraUploadSyncLock)
-            {
-                ContentUploadHistory history;
-
-                try
-                {
-                    history = _json.DeserializeFromFile<ContentUploadHistory>(path);
-                }
-                catch (IOException)
-                {
-                    history = new ContentUploadHistory
-                    {
-                        DeviceId = deviceId
-                    };
-                }
-
-                history.DeviceId = deviceId;
-
-                var list = history.FilesUploaded.ToList();
-                list.Add(file);
-                history.FilesUploaded = list.ToArray();
-
-                _json.SerializeToFile(history, path);
-            }
-        }
-
-        internal Task EnsureLibraryFolder(string path, string name)
-        {
-            var existingFolders = _libraryManager
-                .RootFolder
-                .Children
-                .OfType<Folder>()
-                .Where(i => _fileSystem.AreEqual(path, i.Path) || _fileSystem.ContainsSubPath(i.Path, path))
-                .ToList();
-
-            if (existingFolders.Count > 0)
-            {
-                return Task.CompletedTask;
-            }
-
-            Directory.CreateDirectory(path);
-
-            var libraryOptions = new LibraryOptions
-            {
-                PathInfos = new[] { new MediaPathInfo { Path = path } },
-                EnablePhotos = true,
-                EnableRealtimeMonitor = false,
-                SaveLocalMetadata = true
-            };
-
-            if (string.IsNullOrWhiteSpace(name))
-            {
-                name = _localizationManager.GetLocalizedString("HeaderCameraUploads");
```

Comment:
I think the "HeaderCameraUploads" translation string can be removed from the translation files now.

Reply:
I think that should be left to a separate pr. Removing everything I have so far is already pretty far out of scope, and I think that would be enough to justify its own pr

Reply:
Sounds good, I would just open a new issue to make sure it is tracked and not forgotten about (unless you're going to open a separate PR yourself immediately instead)